frontend/workflows/projectSelector/src/project-group.tsx

```diff
@@ -140,7 +140,7 @@ const ProjectGroup: React.FC<ProjectGroupProps> = ({ title, group, displayToggle
             onChange={() =>
               dispatch({
                 type: "TOGGLE_ENTIRE_GROUP",
-                payload: { group },
+                payload: { group, projects: Object.keys(state[group]) },
```

Comment:
since we may sometimes render out filtered state data, we should only toggle what's shown in the group list. without it, we toggle the status of hidden objects since we currently toggle everything that is in the original state